### ISSUE TITLE
Opening the popup using alt/option + down arrow doesn't focus the popup when using NVDA

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4884]
### Description

When opening the popup of a component with Alt (Windows)/Option (Mac) + Down Arrow, the focus does not move to the popup in Chrome. In Firefox, the focus may move only when one segment is focused.

If all segments are focused (for example, when tabbing into the component for the first time) and the popup is opened, the date inside the popup does not receive focus. Sometimes, the focus moves randomly after opening and closing the popup multiple times.

### Steps To Reproduce

Install NVDA and open one of our online [examples](https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/keyboard-navigation). Focus the input of the specific component and use Alt/Option + down arrow and try to navigate the popup using the Calendar navigation.

### Screenshots or video

Chrome:

https://github.com/user-attachments/assets/439bf073-f46d-414b-83e8-bf39ac91b3e6

Firefox:

https://github.com/user-attachments/assets/70a33d6f-9478-4572-9799-1777dafa3001

## Workaround

Programmatically focus the calendar after the popup is opened:

https://stackblitz.com/edit/angular-5cmnavze

Workaround with a directive:

https://stackblitz.com/edit/angular-5cmnavze-tclpxk8q
